reactive/etcd.py

```diff
@@ -189,6 +194,7 @@ def send_cluster_connection_details(cluster, db):
 
 @when('db.connected')
 @when('etcd.ssl.placed')
+@when_not('cluster.joined')
```

Comment:
The important part is here, this is the inverse of the previous method. I think this is good!